cinema_service/settings.py

```diff
@@ -10,19 +10,20 @@
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 
+from datetime import timedelta
 from pathlib import Path
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+SECRET_KEY = ("django-insecure-"
+              "6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
The SECRET_KEY is hardcoded and should not be exposed in the codebase, especially for production environments. Consider using environment variables to manage sensitive information.